target/linux/realtek/files-5.15/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -749,7 +747,7 @@ static int rtl8380_configure_int_rtl8218b(struct phy_device *phydev)
 	// }
 
 	val = phy_read(phydev, 0);
```

Comment:
This can be also replaced with `MII_BMCR`.

I actually took a good look at this file, and found that there are too many magic numbers which can be replaced with standard macros. May use a separate commit.

Reply:
Thanks a lot for the hint, I've now also replaced the `0` with `MII_BMCR` in all other occurences of `phy_read(phydev, 0)` and `phy_write(phydev, 0, val)`.